### DISCUSSION TITLE
QGIS script  fails in mainland China.（with VPN）

### DISCUSSION THREAD

**Discussion by BobNJU** [giswqs/qgis-earthengine-examples#22] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/30338266/163591122-43b1ff28-510d-483a-90d5-4c03e0a7d943.png)
Hello professor, I failed to run the tutorial demo. How can I solve it? Thank you!

**Comment by giswqs** [comment, marked as answer]
Not all VPN will work. See https://zhuanlan.zhihu.com/p/269419231.

**Reply by BobNJU** [reply]
Professor Wu is as helpful as ever. I've only set up the proxy on notebook with your geemap before, but now I've learned the OS library Settings. Thank you for your advice. Can I buy you some coffee?

**Reply by giswqs** [reply]
Glad to hear that it worked for you. Thank you for the coffee. Appreciated.
